### PR TITLE
Change monitoring system to stacklight in mk22_lab_basic

### PR DESCRIPTION
**Pull request by dkalashnik**

### CODE REVIEW

File: classes/cluster/mk22_lab_basic/fuel/config.yml [hunk 1]
```diff
@@ -3,28 +3,24 @@ classes:
 - system.linux.system.single
 - system.linux.system.repo.tcp_salt
 - system.openssh.client.lab
-- system.salt.master.pkg
+- system.salt.master.git
 - system.reclass.storage.salt
 - system.salt.minion.pki.authority
 - system.salt.minion.pki.certificate
 - system.sphinx.server.doc.reclass
-- system.keystone.client.single
```
Comment (elemoine) [inline classes/cluster/mk22_lab_basic/fuel/config.yml]: Any reason for removing this class? It is in cluster/mk20_stacklight_basic/fuel/config.yml so I am wondering…

File: classes/cluster/mk22_lab_basic/fuel/config.yml [hunk 2]
```diff
@@ -3,28 +3,24 @@ classes:
...
-- system.keystone.client.service.ceilometer
-- system.keystone.client.service.nova21
-- system.mysql.client.single
```
Comment (elemoine) [inline classes/cluster/mk22_lab_basic/fuel/config.yml]: Same comment. Do you really to remove this class? And note that cluster/mk20_stacklight_basic/fuel/config.yml has system.keystone.client.service.nova.

File: classes/cluster/mk22_lab_basic/openstack/compute.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ classes:
 - system.linux.system.repo.mos9
 - system.nova.compute.cluster
 - system.opencontrail.compute.cluster
+- service.opencontrail.compute.cluster
```
Comment (elemoine) [inline classes/cluster/mk22_lab_basic/openstack/compute.yml]: What about system.heka.alarm.openstack_compute? See https://github.com/Mirantis/mk-lab-salt-model/blob/master/classes/cluster/mk20_stacklight_basic/openstack/compute.yml#L6.

File: classes/cluster/mk22_lab_basic/openstack/control.yml [hunk 4]
```diff
@@ -17,6 +17,7 @@ classes:
 - system.galera.server.database.ceilometer
 - system.galera.server.database.cinder
 - system.galera.server.database.glance
+- system.galera.server.database.grafana
 - system.galera.server.database.heat
 - system.galera.server.database.keystone
 - system.galera.server.database.nova
```
Comment (elemoine) [inline classes/cluster/mk22_lab_basic/openstack/control.yml]: What about system.heka.alarm.openstack_control? See https://github.com/Mirantis/mk-lab-salt-model/blob/master/classes/cluster/mk20_stacklight_basic/openstack/control.yml#L23.

Comment (dkalashnik) [inline classes/cluster/mk22_lab_basic/openstack/control.yml]: Done

File: classes/cluster/mk22_lab_basic/openstack/compute.yml (hunk 3 above)
Comment (dkalashnik) [inline classes/cluster/mk22_lab_basic/openstack/compute.yml]: done

File: classes/cluster/mk22_lab_basic/fuel/config.yml [hunk 5]
```diff
@@ -3,28 +3,24 @@ classes:
 - system.linux.system.single
 - system.linux.system.repo.tcp_salt
 - system.openssh.client.lab
-- system.salt.master.pkg
+- system.salt.master.git
```
Comment (dkalashnik) [inline classes/cluster/mk22_lab_basic/fuel/config.yml]: @elemoine @cznewt 
We have to decide: are we really going to change that? In mk22_lab_advanced the same changes required.
I assume that we should have 2 sets of model, prod which is for production, based on packages and development, git-based, for a quick updates and testing of not released features.

**Comment by dkalashnik** [comment]
@elemoine @cznewt 
Lets get it merged, I've greverted change from pkg to git formulas

File: classes/cluster/mk22_lab_basic/stacklight/client.yml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+classes:
+- system.collectd.client.output.heka
+- system.heka.log_collector.single
+- system.heka.metric_collector.single
+- cluster.mk22_lab_basic.stacklight
```
Comment (cznewt) [inline classes/cluster/mk22_lab_basic/stacklight/client.yml]: leave this cluster class [it goes from cluster.init], otherwise looks good

Comment (dkalashnik) [inline classes/cluster/mk22_lab_basic/stacklight/client.yml]: Done,
All changes since previous version: http://pastebin.com/J6PEP9dK

**Pull request merged**